### DISCUSSION TITLE
Is `wants` a valid option??

### DISCUSSION THREAD

**Discussion by amirhhashemi** [wbthomason/packer.nvim#871] [Q&A] [answered]
I've seen some people use an option named `wants` (for example, [in the wiki](https://github.com/wbthomason/packer.nvim/wiki/Tips-and-Tricks#utility-function-to-specify-many-key-combinations)) but I can't find any documentation for that. Is it a valid option?

**Comment by amirhhashemi** [comment, marked as answer]
I found some PRs about adding documentation for `wants`, so I assume it's a valid option.

**Reply by nyngwang** [reply]
So why it's not documented? Could you provide the link to the PR?

**Reply by amirhhashemi** [reply]
I'm not sure but I think it's not documented to prevent people from using it. Apparently, it's a temporary fix for an issue with lazy loading. At least this is what I gathered from these two links:

https://github.com/wbthomason/packer.nvim/pull/264
https://github.com/wbthomason/packer.nvim/issues/537

Sorry for not providing the links in the answer.

**Reply by nyngwang** [reply]
No worries. I did find these two issues according to your hints("I found some **PRs**") and have already left some comments there.
